templates/home.html

```diff
@@ -75,6 +75,14 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
             <span class="ml small">അറിയിപ്പുകൾ</span>
         </span>
     </a>
+    <a href="https://www.facebook.com/IndiaHSI/" class="home-button card" role="button">
+        {% bootstrap_icon "knight" %}
+        <span class="text">
+        Animal Rescue<br/>
+            <span class="ml small notranslate">മൃഗ രക്ഷ സഹായം അഭ്യര്‍ഥിക്കാന്‍ </span><br/>
+			<span class="ml small notranslate">Helpline: +91 89433 22808 </span>
```

Comment:
It would be great if you could add a `<a>` tag as well to the phone number so that folks can directly make a call instead of copying and pasting the number to dialer.
Simply replace `+91 89433 22808` to `<a href='tel: +91 89433 22808'>+91 89433 22808</a>`.

Reply:
This is not going to work, the whole card is wrapped by a <a>, so added another <a> inside that won't work in this case.

Reply:
@npari remove the parent `<a>` tag and wrap all content other than the phone number with that `href`. Then the `<a>` tag on the phone number will work.

Reply:
We need the card to redirect to the animal rescue page.